scripts/storage/secondary/createtmplt.sh

```diff
@@ -210,7 +210,6 @@ echo "filename=$tmpltname" > /$tmpltfs/template.properties
 echo "description=$descr" >> /$tmpltfs/template.properties
 # we need to rethink this property as it might get changed after download due to decompression
 # option is to recalcutate it here
-echo "checksum=$cksum" >> /$tmpltfs/template.properties
```

Comment:
Why is this line being deleted from the unrelated file?

Reply:
in my first PR i tried to remove the "createtmplt.sh" file completely from creation but it is "ugly" in java code used. by this way i found an not used variable. i think checksum calculation was removed from Dan  Hoogland #2246. it was forgotten. i can make a seperate PR if this is better?